static/dbstructure.config.php

```diff
@@ -1296,24 +1296,26 @@
 		"comment" => "tags and mentions",
 		"fields" => [
 			"id" => ["type" => "int unsigned", "not null" => "1", "extra" => "auto_increment", "primary" => "1", "comment" => ""],
-			"type" => ["type" => "tinyint unsigned", "not null" => "1", "default" => "0", "comment" => ""],
```

Comment:
What is your reason for moving the type away from the `tag` table? If I understand correctly, the `url` is heavily related to the `type` of the term.

Reply:
The type moved to the other table since we now simply store the contact id if we have one. So we don't always have got a `tag` entry.

Reply:
But `tid` is `NOT NULL`? Also do we still need URL then?

Reply:
We need the `url` for all contacts that hadn't been found in the system. Concerning the `tid` I entered a default value now. Seems like you cannot set a primary key field to `null`

Reply:
Of course not! It defeats the point of the index key in the first place.